R/backport_linter.R

```diff
@@ -45,12 +45,13 @@ backport_linter <- function(r_version = getRversion(), except = character()) {
   backport_index <- rep(names(backport_blacklist), times = lengths(backport_blacklist))
   names(backport_index) <- unlist(backport_blacklist)
 
-  names_xpath <- "//SYMBOL | //SYMBOL_FUNCTION_CALL"
-
   Linter(linter_level = "expression", function(source_expression) {
     xml <- source_expression$xml_parsed_content
 
-    all_names_nodes <- xml_find_all(xml, names_xpath)
+    all_names_nodes <- combine_nodesets(
```

Comment:
might suggest this as a helper for {xml2} directly, it seems generally useful

Reply:
I'd suggest `c.xml_nodeset` to be implemented. No need for a custom function IMO.

Reply:
https://github.com/r-lib/xml2/issues/433

Reply:
Great. If not, we might want to export our helper to facilitate downstreams using it since it comes up a fair amount in our suite.